xt/41-coaload.t

```diff
@@ -1,6 +1,7 @@
 use Test::More;
 use strict;
 use warnings;
+use LedgerSMB::Sysconfig;
```

Comment:
This isn't completed. We assume that the database is always on the LSMB machine. Environment variables used for tests specify the database name, but we should get PG address from Sysconfig.